### PR TITLE
PD-6313 update templates/ fix linting errors

### PR DESCRIPTION
**Pull request by sathvik-dhanya**
Updated templates to reflect what's in our local charts
Also trying to fix linting errors

`namerd-service.yaml` file name has been fixed.

### CODE REVIEW

File: templates/namerctl-job.yaml [hunk 1]
```diff
@@ -10,20 +10,20 @@ spec:
...
-            - "/namerctl/createNs.sh"
-          volumeMounts:
-            - name: "namerctl-script"
-              mountPath: "/namerctl"
-              readOnly: true
+      - name: namerctl
+        image: linkerd/namerctl:0.8.6
+        env:
+        - name: NAMERCTL_BASE_URL
+          value: http://namerd.infra.svc.cluster.local:4180
```
Comment (wolf31o2) [inline templates/namerctl-job.yaml:22]: This URL needs to take into account the actual namespace we're deployed into.

File: templates/namerd-admin-service.yaml [hunk 2]
```diff
@@ -2,22 +2,16 @@
...
-    - name: thrift
-      port: 4100
-    - name: http
-      port: 4180
-    - name: mesh
-      port: 4321
-    - name: admin
-      port: 9991
+  - name: admin
+    port: 9991
```
Comment (wolf31o2) [inline templates/namerd-admin-service.yaml:17]: Completely drop this service. I created it, initially, to give a different type to just the admin interface. I'm not sure we need this in a public chart, at all.

File: templates/namerctl-job.yaml (hunk 1 above)
Comment (sathvik-dhanya) [reply templates/namerctl-job.yaml:22]: Currently the namespace is hard coded to `infra` in all the namerd charts.. so I did not change this URL.
But I can make the namespace configurable. That'd be better?

Comment (wolf31o2) [reply templates/namerctl-job.yaml:22]: That's fine. Leave this. We can make the namespace configurable in another PR which does just that.

**Pull request merged**